### PR TITLE
Mint signup form

### PR DESCRIPTION
**Pull request by m15e**
Add header
Add form
Add footer

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  margin-left: 6px;
+  display: flex;
+  font-size: 16px;
+  align-items: center;
+  justify-content: center;
+}
+
+#privacy-statement {
+  font-style: italic;
+  font-size: 12px !important;
```
Comment (andresporras3423) [inline style.css]: Don't use !important

File: style.css [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+#signup-form-captcha p {
+  padding: 10px;
+}
+
+#signup-form-captcha a,
+#signup-form-captcha p {
+  font-size: 10px !important;
+}
+
+#privacy {
```
Comment (andresporras3423) [inline style.css]: Don't overuse ID selectors

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @m15e  and @Joseph-Burke** 
You did a great job with your first submission for this project, your page looks really close to the original  :clap: :clap:
There are still some improvements to do:

- [x] Don't use !important
- [x] Don't overuse ID selectors
- [ ] Next time try to make a more meaningful short description of your pull request
- [x] The password field of your form should have an open lock icon :point_down:


### OPTIONAL

- [x] it's recommended to use your footer tag after your main tag, not inside

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: style.css (hunk 2 above)
Comment (Joseph-Burke) [reply style.css]: Hey Oscar, thanks for the great review, it has helped a lot and we've implemented the improvements you suggested.

We were discussing ID vs class selectors and I was wondering if you could help us better understand when ID selectors are appropriate and when class selectors are. Should we always go for class selectors first when possible?

Joe

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    <label>
+      Phone (recommended)
+      <input type='tel' name="phone">
+      <sub>Standard call, messaging or data rates may apply.</sub>
+    </label>
+    <label>
+      Password
+      <input type='password' name="password">
+    </label>
+    <button><img src="assets/lock.svg" alt='lock icon'> Create Account</button>
```
Comment (HADDADSOHAIB) [inline index.html]: The button should have a type='submit':
- [x] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @m15e and @Joseph-Burke     :handshake: 

### You did wonderful job :clap:  your project is looking good :medal_sports: and look like the original. 

### There are some changes to get the project approved :point_down:
- You used a picture for this part:
Screenshot from 2020-06-05 12-48-40
You should only use pictures for logos, not text, you need to build the structure of that part using HTML and CSS.
##### Intuit products
- [x] The products row is under the Intuit logo and above the form ([link to page](https://accounts.intuit.com/signup.html))
- [x] It shows the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [x] It shows the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [x] It shows the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
You should always follow the original, on the mint website they are not using an image for the whole part.
- Add this file to gitignore:
Screenshot from 2020-06-05 12-45-34
- Here is some articles that could help you understand when to use classes and ids in css:
https://css-tricks.com/the-difference-between-id-and-class/
- check comments for more details.
### After implementing the changes, please ask for another code review.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi, @m15e and @Joseph-Burke, great job with this project :+1: Having fixed all the changes identified by the previous TSE's, I hereby approve your project :slightly_smiling_face: . However, there is an important point to note. There is a horizontal scroll that needs to be fixed, for example; by setting the width to 100%

Go ahead and merge to master and submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**